src/SWP/Component/Common/Generator/RandomStringGenerator.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Common Component.
+ *
+ * Copyright 2015 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+namespace SWP\Component\Common\Generator;
+
+class RandomStringGenerator implements GeneratorInterface
```

Comment:
we can generate whatever code length we want, we dont need over 10 chars codes which uniqid generates. This way we can customize it by having a custom generator.